docs/tutorial.rst

```diff
@@ -2,14 +2,19 @@
 Tutorial
 ========
 
-This tutorial is for macOS. Linux and Windows are not yet officially supported but should work for the most part.
+This tutorial is for macOS and Windows. Linux is not yet officially supported but should work for the most part.
```

Comment:
i don't think we should say Windows is "officially supported" since we don't actually support anybody on the team on win and we don't have a test VM yet.